tests/ast/data/df_analytics_functions.test

```diff
@@ -57,16 +57,16 @@ time_series_agg_res = df.analytics.time_series_agg(
 
 df = session.create_dataframe([["2023-01-01", 101, 200], ["2023-01-02", 101, 100], ["2023-01-03", 101, 300], ["2023-01-04", 102, 250]])
 
-res1 = df.to_df("ORDERDATE", "PRODUCTKEY", "SALESAMOUNT")
+df = df.to_df("ORDERDATE", "PRODUCTKEY", "SALESAMOUNT")
```

Comment:
Are you sure you're using the current unparser?

Reply:
Oh, I think this PR might include all of the test updates from your addition of expression inlining to the Unparser now, is that okay?

Reply:
updated with recent main, I think this should be not relevant anymore.